src/renderers/shared/fiber/ReactFiberCommitWork.js

```diff
@@ -302,10 +307,22 @@ module.exports = function<T, P, I, TI, C, CX>(
   }
 
   function commitDeletion(current : Fiber) : void {
-    // Recursively delete all host nodes from the parent.
-    const parent = getHostParent(current);
-    // Detach refs and call componentWillUnmount() on the whole subtree.
-    unmountHostComponents(parent, current);
+    // Recursively delete the closest child host nodes from the closest host parent.
+    // Then detach refs and call componentWillUnmount() on the whole subtree.
+    if (current.tag === HostPortal) {
+      // When deleting a portal, there are no host nodes above it.
+      // It is a host parent itself.
```

Comment:
I could probably inline it there if we know for sure that portal can never be tagged with a deletion effect (can it not? I don’t understand that part well)